### ISSUE TITLE
Google Summer of Code project?

### ISSUE THREAD

**Issue by Gabriella439** [haskell-crypto/cryptonite#306]
This is primarily addressed to @ocheron since I wasn't sure how else to get in touch.

Would you or any other maintainer of this project be interested in mentoring a Google Summer of Code student to contribute to this project?  A beginner Haskell programmer with experience in networking and cryptography reached out to me interested in participating in the program and I'm trying to match them up with a suitable project to contribute to.  I can provide more details (including their resume) by e-mail.

**Comment by ocheron** [comment]
I contribute just as hobbyist here.
Maybe someone with experience in the field will read your message.

**Comment by Gabriella439** [comment]
@ocheron: The main requirements are:

* You submit an idea for a useful project that would be suitable for a student to work on for three months
* You're available to mentor the project, which is typically a time commitment of ~5 hours a week (based on my own experience), which includes coaching the student contribute to the project effectively and reviewing their pull requests
* You have authority to approve/merge their work (so the student doesn't run the risk of working on something that won't be accepted)

So being a hobbyist is not disqualifying.  In fact, Google Summer of Code is an opportunity to pay a student to work full time on something that you might not have free time to do so yourself.

The main reason I pinged you specifically is because I looked at the commit history for `cryptonite` and saw that you had made most of the recent contributions.
